omisego-sdk/src/main/java/co/omisego/omisego/websocket/SocketCallback.kt

```diff
@@ -0,0 +1,84 @@
+package co.omisego.omisego.websocket
+
+/*
+ * OmiseGO
+ *
+ * Created by Phuchit Sirimongkolsathien on 8/5/2018 AD.
+ * Copyright © 2017-2018 OmiseGO. All rights reserved.
+ */
+
+import co.omisego.omisego.model.APIError
+import co.omisego.omisego.model.socket.SocketReceive
+import co.omisego.omisego.model.transaction.consumption.TransactionConsumption
+
+/**
+ * A web socket connection callback that executed when the web socket client is connected to the server or disconnected from the server.
+ */
+interface SocketConnectionCallback {
+    /**
+     * Invoked when the web socket client has connected to the eWallet web socket API successfully.
+     */
+    fun onConnected()
+
+    /**
+     * Invoked when the web socket client has disconnected from the eWallet web socket API.
+     *
+     * @param throwable (Optional) The exception might be raised if the web socket was not disconnected successfully.
+     */
+    fun onDisconnected(throwable: Throwable?)
+}
+
+interface SocketChannelCallback {
+    /**
+     * Invoked when the client have been joined the channel successfully.
```

Comment:
Typo, *Invoked when the client joined the channel successfully* would be correct :)